UI/CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ set_target_properties(
   PROPERTIES AUTOMOC ON
              AUTOUIC ON
              AUTORCC ON
+             AUTORCC_OPTIONS "--format-version;1"
```

Comment:
```suggestion
             AUTORCC_OPTIONS "--format-version;1;--compress-algo;zlib"
```

Reply:
Context: obs-deps build Qt only with zlib but Linux have all features and then default to zstd which does not work with version less than 3.

Reply:
Yeah, that works. Apparently Qt also supports `SOURCE_DATE_EPOCH` nowadays to force a specific timestamp, but setting that from within cmake did not appear to work for Windows so I'm sticking with overriding the version for now.